src/main/features/first-contact/paths.ts

```diff
@@ -8,4 +8,5 @@ export class Paths {
 
 export class ErrorPaths {
   static readonly claimSummaryAccessDeniedPage = new RoutablePath('/first-contact/access-denied')
+  static readonly ccjRequestedHandoffPage = new RoutablePath('/first-contact/claimant-requested-for-ccj')
```

Comment:
this doesn't read nicely to me: `claimant-requested-for-ccj` 
how about:
`claimant-already-requested-ccj`
or
`claimant-has-requested-ccj`

Reply:
I just took it from your suggestion earlier. Ill update now

Reply:
I meant the page url btw on this one, not the variable